vmdb/app/models/mixins/ems_openstack_mixin.rb

```diff
@@ -117,6 +117,22 @@ def stack_create(stack_name, template, options = {})
     raise MiqException::MiqOrchestrationProvisionError, err.to_s, err.backtrace
   end
 
+  def stack_update(stack_name, stack_id, options = {})
+    stack = openstack_handle.orchestration_service.stacks.get(stack_name, stack_id)
+    stack.save(options) if stack
```

Comment:
Why is this one calling `save` when the one in `ems_amazon` is calling `update`?